core/src/main/scala/com/softwaremill/kmq/redelivery/streams/StartMarkerStream.scala

```diff
@@ -0,0 +1,38 @@
+package com.softwaremill.kmq.redelivery.streams
+
+import akka.Done
+import akka.actor.ActorSystem
+import akka.kafka.ConsumerMessage.CommittableMessage
+import akka.kafka.scaladsl.Consumer.DrainingControl
+import akka.kafka.scaladsl.{Committer, Consumer, Producer}
+import akka.kafka._
+import akka.stream.scaladsl.Sink
+import com.softwaremill.kmq.{MarkerKey, MarkerValue, StartMarker}
+import org.apache.kafka.clients.producer.ProducerRecord
+
+class StartMarkerStream(msgConsumerSettings: ConsumerSettings[String, String],
+                        markerProducerSettings: ProducerSettings[MarkerKey, MarkerValue],
+                        queueTopic: String, markersTopic: String, maxPartitions: Int, messageTimeout: Long)
+                       (implicit system: ActorSystem) {
+
+  def run(): DrainingControl[Done] = {
```

Comment:
This should be wrapped with RestartSource: https://doc.akka.io/docs/akka/current/stream/stream-error.html#delayed-restarts-with-a-backoff-operator not sure if here or at a different level

Reply:
Okay, StartMarkerStream source surrounded with RestartSource.withBackoff()

Reply:
After discussion with @adamw the `StartMarkerStream` appears unnecessary.

Reply:
I've removed StartMarkerStream

Reply:
It's possible that I'll introduce something similar in the HTTP API